### DISCUSSION TITLE
Truecharts/Truenass Install

### DISCUSSION THREAD

**Discussion by DevilsDesigns** [MDeLuise/plant-it#159] [Q&A]
I was wondeirng if there was a way to add this to truecharts or truenas scale cobia.  Apparently you can install a custom app but it only allows for one repo to pull from docker. Any ideas or plans to port this to truenas truecharts or truenas itself. https://www.truenas.com/docs/scale/23.10/scaleuireference/apps/installcustomappscreens/

**Comment by MDeLuise** [comment]
Hi @DevilsDesigns,

Thanks for reaching out. The charts for Plant-it [are now available within TrueCharts](https://truecharts.org/charts/incubator/plant-it/).

Please note that I am not the creator of these charts. For any issues or detailed instructions on how to deploy using TrueCharts, please refer to the [official TrueCharts documentation](https://truecharts.org/guides/).

Also, I wanted to mention that I create Helm charts for the app, and you can find instructions on how to use them [here](https://docs.plant-it.org/0.7.1/server-installation/#kubernetes-deployment). So besides the charts provided by TrueCharts, you also have the option to use directly provided charts.
